api/tests/mocha/services/setup/view-indexer.spec.js

```diff
@@ -188,8 +188,9 @@ describe('View indexer service', () => {
 
       viewIndexer.__set__('continueIndexing', true);
       const nextTick = () => new Promise(r => setTimeout(r));
-      const timeoutFn = () => new Promise((resolve, reject) =>
-        setTimeout(() => reject({ error: { code: 'ESOCKETTIMEDOUT' } }))
+      const timeoutFn = () => new Promise(
+        (resolve, reject) =>
+          setTimeout(() => reject({ error: { code: 'ESOCKETTIMEDOUT' } }))
```

Comment:
I'm not a fan of multi-line auto return functions at the best of times, but this one has two! The newline forced by the new eslint rule makes it even harder to read IMO. I don't think it's the rule at fault just wanted to point it out. IMO this is more readable...

```suggestion
      const timeoutFn = () => {
        return new Promise((resolve, reject) => {
          setTimeout(() => reject({ error: { code: 'ESOCKETTIMEDOUT' } }));
        });
      };
```